sample/src/main/res/drawable/ic_remove.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#FFFFFF"
```

Comment:
Maybe you can rename this file as `ic_subtract` instead? Because the button's purpose is to subtracting, not removing the number. 

Or maybe it's better if the name follows the views ids as below:
- ic_add.xml -> ic_increment.xml
- ic_remove.xml -> ic_decrement.xml